.gitignore

```diff
@@ -22,4 +22,6 @@ yarn.lock
 # IDEA
 *.iml
 *.ipr
-*.iws
\ No newline at end of file
+*.iws
+
+\demo
```

Comment:
@SchwarzerYV  ？Are you sure this needs to be modified?

Reply:
Oh, it’s my mistake, they shouldn't be modified